Since this fontconfig change [1], we get warnings in 44-wqy-zenhei.conf
like this:   "Having multiple values in <test> isn't supported and may 
not work as expected" - let's fix that...

[1] http://cgit.freedesktop.org/fontconfig/commit/?id=2837c63876b9b

diff -Nur wqy-zenhei.orig/44-wqy-zenhei.conf wqy-zenhei/44-wqy-zenhei.conf
--- wqy-zenhei.orig/44-wqy-zenhei.conf	2009-05-17 15:25:41.000000000 -0500
+++ wqy-zenhei/44-wqy-zenhei.conf	2013-02-25 19:00:36.870491434 -0600
@@ -6,7 +6,11 @@
   <match target="font">
         <test qual="any" name="family">
                 <string>WenQuanYi Zen Hei</string>
+        </test> 
+        <test qual="any" name="family">
                 <string>文泉驿正黑</string>
+        </test> 
+        <test qual="any" name="family">
                 <string>文泉驛正黑</string>
         </test> 
         <edit name="globaladvance"><bool>false</bool></edit>
